src/main/java/com/flansmod/common/driveables/EntityDriveable.java

```diff
@@ -1,2809 +1,233 @@
-package com.flansmod.common.driveables;
+package com.flansmod.common.teams;
```

Comment:
Something has gone seriously wrong here, why has EntityDriveable become the teams manager?